Link error when including HytecMotor support in a Windows IOC

The following link errors occur when when including HytecMotor support in a Windows IOC (EPICS_HOST_ARCH=windows-x64-static, VS 2015):
```
iockmp2Win64_registerRecordDeviceDriver.obj : error LNK2019: unresolved external symbol pvar_func_Hytec8601Register referenced in function iockmp2Win64_registerRecordDeviceDriver
```

--- a/motorApp/HytecSrc/HytecMotorDriver.cpp
+++ b/motorApp/HytecSrc/HytecMotorDriver.cpp
@@ -1067,6 +1067,9 @@ extern "C" int Hytec8601Configure(const char *portName,
     return(asynSuccess);
 }
 
+extern "C"
+{
+
 static const iocshArg Hy8601ConfigArg0 = {"portName", iocshArgString};
 static const iocshArg Hy8601ConfigArg1 = {"numAxes", iocshArgInt};
 static const iocshArg Hy8601ConfigArg2 = {"movingPollPeriod", iocshArgInt};
@@ -1110,4 +1113,4 @@ static void Hytec8601Register(void)
 
 epicsExportRegistrar(Hytec8601Register);
 
-
+}

